src/docs/reference/modules/Migrations/README.md

```diff
@@ -1,9 +1,11 @@
 # Data Migrations (`OrchardCore.Data.Migration.DataMigration`)
 
+## Configuring Data Migration
+
 Data Migration classes can be used to alter the content type definitions (like by adding new __types__, or configuring their __parts__ and __fields__), 
 initializing recipes or creating indices.
 
-There should be one Migrations file per module inheriting from __DataMigration__. However, if a module has more than one feature and multiple migrations per feature, each migration should be decorated with the Feature attribute i.e. `[Feature("OrchardCore.ContentFields.Indexing.SQL")]`
+There should be one Migrations file per module inheriting from __DataMigration__. However, if a module has more than one feature, it can be added one migration file per feature, being decorated with the Feature attribute i.e. `[Feature("OrchardCore.ContentFields.Indexing.SQL")]`
```

Comment:
The docs seems to be outdated event with the suggested test.  Please remove this paragraph altogether.